src/main/scala/nl/knaw/dans/easy/fsrdb/Conf.scala

```diff
@@ -27,27 +27,27 @@ class Conf(args: Seq[String], props: PropertiesConfiguration) extends ScallopCon
   banner(s"""
             | Update the EASY File-system RDB with data about files and folders from the EASY Fedora Commons Repository.
             |
-            | Usage: $printedName
-            |              [-f <fcrepo-server>] \\
-            |              [-u <fcrepo-user> \\
-            |               -p <fcrepo-password>] \\
-            |              [-d <db-connection-url>] \\
-            |               <dataset-pid>
+            | Usage: $printedName [<option>...] [--file <text-file-with-dataset-id-per-line> | <dataset-pid>...]
+            |
             | Options:
             |""".stripMargin)
-  val fedora = opt[URL]("fcrepo-server",
-    descr = "Fedora Commons Repository Server to connect to ",
+  val fedora = opt[URL]("fcrepo-server", short = 'f',
+    descr = "URL of the Fedora Commons Repository Server to connect to ",
     default = Some(new URL(props.getString("default.fcrepo-server"))))
-  val user = opt[String]("fcrepo-user",
+  val user = opt[String]("fcrepo-user", short = 'u',
     descr = "User to connect to fcrepo-server",
     default = Some(props.getString("default.fcrepo-user")))
-  val password = opt[String]("fcrepo-password",
+  val password = opt[String]("fcrepo-password", short = 'p',
     descr = "Password for fcrepo-user",
     default = Some(props.getString("default.fcrepo-password")))
-  val db = opt[String]("db-connection-url",
-    descr="JDBC connection URL to File-system RDB (including user and password parameters)",
+  val db = opt[String]("db-connection-url", short = 'd',
+    descr="URL of the JDBC connection to File-system RDB (including user and password parameters)",
     default = Some(props.getString("default.db-connection-url")))
-  val dataset = trailArg[String]("dataset-pid",
-    descr = "Dataset for which to update the file and folder metadata in the File-system RDB",
-    required = true)
+  val input = opt[File]("file",
```

Comment:
A bit puzzled. Do you mean that Scallop will turn it automatically into a `trailArg` or that Paul should turn it into a `trailArg` and that Scallop will be clever enough to handle that?